vendor/engines/projects/app/controllers/projects/projects_controller.rb

```diff
@@ -83,6 +88,10 @@ def showing_inactive?
 
     private
 
+    def initial_projects
+      nil
```

Comment:
Any reason to use `nil` here instead of an empty array?  curious why this needs a method of its own?  maybe the searcher should have this as an optional input?

Reply:
The searcher uses recursion to filter the previous list of projects, making the input optional would also mean switching the params. I can do that but I wonder if it's the best as it'd make this searcher different from the rest and could be a bit confusing.

It definitely doesn't need to be its own method, I'll fix that.